content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration/starshipit_shipping.rst

```diff
@@ -0,0 +1,201 @@
+  .
+- Manually :guilabel:`Save` the form by clicking the cloud icon next to the :guilabel:`Shipping
+  Methods / New` breadcrumbs.
+
+After configuring and saving the form, follow these steps to load the shipping products:
+
+- In the :guilabel:`Starshipit Configuration` tab of the :guilabel:`New Shipping Method` form, click
+  on the :guilabel:`Select a service linked to your Starshipit account` link.
+- Select the shipping service to be used for deliveries and returns.
+- Click :guilabel:`Confirm`.
+
+.. example::
+   Sample of a Starshipit shipping product configured in Odoo:
+
+   | :guilabel:`Sendle: Sendle drop off`
+   | :guilabel:`Shipping Product`: `Sendle Delivery`
+   | :guilabel:`Starshipit Service Code`: `STANDARD-DROPOFF`
+
+.. image:: starshipit_shipping/starshipit-configuration.png
+   :align: center
+   :alt: Example of shipping products configured in Odoo.
+
+.. tip::
+   Starshipit does not provide test keys when a company tests the sending of a package in Odoo. This
+   means that if a package is created, your account may be charged.
+
+   Odoo has a built-in layer of protection against unwanted charges when using test environments.
+   Within a test environment, if a shipping method is used to create labels, then those labels are
+   immediately canceled after creation — this occurs automatically. Please note that depending
+   on the shipping provider being used, charges may still occur and you may need to cancel your
+   orders on the provider platform. The test and production environment settings can be toggled back
+   and forth from the :guilabel:`Smart Buttons`.
+
+Generate a label with Starshipit
+--------------------------------
+
+When creating a quotation in Odoo, add shipping and a :guilabel:`Starshipit shipping method`.
+You can get the shipping rate at this point by clicking :guilabel:`Get rate`.
+Odoo's sales apps are fully integrated and our module also works on eCommerce.
+
+Then, :guilabel:`Validate` the delivery. Shipping label documents are automatically generated in the
+chatter, which includes the following:
+
+#. :guilabel:`Shipping label(s)` depending on the number of packages.
+#. :guilabel:`Tracking number(s)` if the selected carrier supports it.
+#. :guilabel:`Return label(s)` if the Starshipit connector is configured for returns.
+
+.. image:: starshipit_shipping/starshipit-shipping.png
+   :align: center
+   :alt: Example of a shipped order in Odoo.
+
+.. important::
+   The package's weight is not automatically verified before sending goods to the shipping provider.
+   Please make sure that you select the correct shipping option for your package, otherwise you
+   may end up with an error.
+   Starshipit verifies the destination address during the order creation too, so make sure it is
+   correct as well.
+   Finally, some couriers may required other information, such as an email address or phone number.
+   Please ensure that all necessary information are set upon sending a shipping order.
```

Comment:
Slight rewrite and add blank spaces between paragraphs to separate them visually.
```suggestion
   Package weight in Odoo is calculated by adding the weights of the products plus the empty package
   saved in the database. Ensure the correct shipping option is selected, as the package weight is
   not automatically verified.

   Verify the destination address, as Starshipit checks it when the order is created.

   Finally, some couriers may require other information, such as an email address or phone number.
   Please ensure that all necessary information are set upon sending a shipping order.
```